src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -389,9 +410,10 @@ export function makeClinicalTrackData(
         = _.groupBy(data,
         isSampleList(cases) ?
             datum=>datum.uniqueSampleKey :
-            datum=>datum.uniquePatientKey);// TS error will be gone when ersin updates API
+            datum=>datum.uniquePatientKey);
 
     // Create oncoprint data
+    const getDataForCase = makeGetDataForCase(attribute, isSampleList(cases) ? "sample" : "patient", data);
```

Comment:
i'd vote for using parens around both the condition and the entire ternary here to make this more readable.

Reply:
around the condition seems excessive... around the ternary I agree. but if you feel strongly I dont care that much

Reply:
ah whatever it does look a little nicer that way